crates/iota-protocol-config/src/lib.rs

```diff
@@ -1632,6 +1640,7 @@ impl ProtocolConfig {
             cfg.vdf_hash_to_input_cost = Some(100);
 
             cfg.feature_flags.passkey_auth = true;
+            cfg.feature_flags.relocate_event_module = true;
```

Comment:
this needs to be enabled at earliest with protocol version 3 in the for-loop bellow

Reply:
I see. Thanks for the comment! Fixed at commit [f60ab7b](https://github.com/iotaledger/iota/pull/4712/commits/f60ab7bc60628b1ed7a111d06fc48ebfe63d0c3a)